src/main/kotlin/lotto/domain/Lottos.kt

```diff
@@ -0,0 +1,29 @@
+package lotto.domain
+
+class Lottos() {
+
+    var lottos = listOf<Lotto>()
+
+    constructor(lottos: List<Lotto>) : this() {
+        this.lottos = lottos
+    }
+
+    fun isInWinningNumber(winningNumbers: List<Int>) {
+        lottos.map {
+            it.countWinningNumber(winningNumbers)
+        }
+    }
+
+    companion object {
+        fun of(purchaseCount: Int): Lottos {
+
+            var list = mutableListOf<Lotto>()
+
+            repeat(purchaseCount) {
+                list.add(Lotto.of())
+            }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            val list = (1..purchaseCount).map { Lotto.of() }
```